papaparse.js

```diff
@@ -1508,7 +1508,17 @@ License: MIT
 				if (duplicateHeaders) {
 					var editedInput = input.split(newline);
 					editedInput[0] = Array.from(headerMap).join(delim);
+					// If we change the size of the input due to duplicate headers
```

Comment:
Could we make the comment smaller? 

What about: `store the number of removed chars to correctly report meta.cursor` ?